pallets/proofs/src/benchmarking.rs

```diff
@@ -0,0 +1,63 @@
+#![cfg(feature = "runtime-benchmarks")]
+
+use frame_benchmarking::v2::*;
+use frame_system::RawOrigin;
+
+use super::*;
+#[allow(unused)]
+use crate::Pallet as ProofsPallet;
+
+// NOTE: these benchmarks are cannot be accurate until the extrinsic takes in a bounded key size
```

Comment:
Let's add a todo/link to an existing issue so we don't forget to do this.

Reply:
this needs a bound, otherwise "prod" compilation fails